src/payment/data/selectors.js

```diff
@@ -90,9 +91,11 @@ export const paymentProcessStatusSelector = state => (state[storeName].basket.pa
  * @param state
  * @return boolean
  * @see POLLING_PAYMENT_STATES
+ * @see WAFFLE_FLAGS.COMMERCE_COORDINATOR_ENABLED
  */
 export const paymentProcessStatusShouldRunSelector = state => (
   POLLING_PAYMENT_STATES.includes(paymentProcessStatusSelector(state))
+  && isWaffleFlagEnabled(WAFFLE_FLAGS.COMMERCE_COORDINATOR_ENABLED)
```

Comment:
The collision of the constant `WAFFLE_FLAGS` with the environmental variable `WAFFLE_FLAGS` set in .env & sister files got confusing for me. What do you think of renaming one of them?

Reply:
Environment is simpler, ill do it there... `ENABLED_WAFFLE_FLAGS`?